src/libertem/io/dataset/dask.py

```diff
@@ -67,7 +67,11 @@ class DaskBackendImpl(MMapBackendImpl):
 
 
 class DaskDataSet(DataSet):
+    _default_min_size = 128 * (2**20)  # MB
+
```

Comment:
I've move the value to a private @property on the class with its own docstring. The reason for making the change was to let a user change the default once-and-for-all rather than embedding it in __init__.

Reply:
Thanks for finding this, though, I didn't know about the interaction with Sphinx/Docstring finding!